xbmc/filesystem/AudioBookFileDirectory.h

```diff
@@ -24,6 +27,9 @@ namespace XFILE
       bool IsAllowed(const CURL& url) const override { return true; }
 
     protected:
+      void AddCommaDelimitedString(const std::vector<std::string>& data,
+                                   const std::vector<std::string>& separators,
+                                   MUSIC_INFO::CMusicInfoTag& musictag);
```

Comment:
static? or probably better in an anonymous namespace of AudioBookFileDirectory.cpp,

Reply:
Don't think there is a need for it to be static or in an anonymous namespace.

Reply:
Why? It is not used outside the cpp and has no state. So, it is a perfect candidate for an implementation detail that not needs to be exposed through the class' interface.